code/modules/vehicles/armored/__armored.dm

```diff
@@ -263,8 +288,14 @@
 			return FALSE
 	return ..()
 
+/obj/vehicle/sealed/armored/attack_ghost(mob/dead/observer/user)
+	. = ..()
+	interior?.mob_enter(user)
```

Comment:
Can you orbit a tank without getting pulled in?

Reply:
fixed to be on mousedrop